thermodynamics/structure_nodes.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
+import numpy as np
+from pyiron_workflow import Workflow
+
+@Workflow.wrap.as_function_node()
```

Comment:
Super minor convenience: when you have nicely named return value(s) (as you do here!) and don't need to specify output channel labels, the last couple versions of pyiron workflow will let you simply invoke `@Workflow.wrap.as_function_node` without the trailing `()` 😄